build.sh

```diff
@@ -27,6 +27,7 @@ PRINT_USAGE() {
     echo "    --icu         : Path to ICU include folder (see example below)"
     echo "    --jobs, -j    : Multicore build (i.e. -j=3 for 3 cores)"
     echo "    --ninja, -n   : Build with ninja instead of make"
+    echo "    --test-build, -t  : Test build (by default Release Build)"
```

Comment:
I think you could update the whole stretch of options here so the colons are aligned.